src/Components/Input/Input.jsx

```diff
@@ -0,0 +1,5 @@
+import "./input.scss";

```

Comment:
Are you ou changing to use PascalCase here, is that the reason for this new file, correct?

Reply:
this input file, was brought by the fetch with the main branch, i forgot to do the force push to update this one.
and, can you give me an explanation on the "PascalCase"

Reply:
But it's not supposed to appear here, because this is a diff between your branch to main, I think that in main branch you're using lower-case 🤔

Reply:
About the pascal case.
Is the way that you will name stuff, check it out:

### TLDR;
- camelCase
- PascalCase
- snake_case
- kebab-case

for more information check this post: https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841